CI doesn't always fail when there is a lint error

http://ci.jujugui.org:8080/job/juju-gui/4158/consoleFull

grep for `node_modules/.bin/eslint` you'll see that there is a lint failure but CI did not fail. 

```
bin/flake8 jujugui
node_modules/.bin/eslint --rulesdir eslint-rules/ jujugui/static/gui/src

/var/lib/jenkins/workspace/juju-gui/jujugui/static/gui/src/app/components/logout/test-logout.js
  26:1  warning  fdescribe() was left in a test  no-single-tests

✖ 1 problem (0 errors, 1 warning)

bin/python setup.py develop
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -16,10 +16,10 @@ rules:
   quotes: [2, "single"]
   strict: [2, "global"]
   no-underscore-dangle: 0
-  no-debugger: 1
+  no-debugger: 2
   no-unused-vars: [2, {args: "none"}]
   indent: [2, 2, {SwitchCase: 1, VariableDeclarator: 2}]
-  no-single-tests: 1
-  no-test-utils: 1
-  react/jsx-key: 1
-  react/jsx-no-duplicate-props: 1
+  no-single-tests: 2
+  no-test-utils: 2
+  react/jsx-key: 2
+  react/jsx-no-duplicate-props: 2

